cfgov/unprocessed/apps/mmt-my-money-calendar/js/index.js

```diff
@@ -1,42 +1,19 @@
-import * as idb from 'idb';
+// import * as idb from 'idb';
 import { render } from 'react-dom';
-import Greeting from './components/greeting';
-import Counter from './components/counter';
+
 import { configure as configureMobX } from 'mobx';
-import { Workbox } from 'workbox-window';
+
+import { StoreProvider } from './stores';
 import Routes from './routes';
-import CashFlowEvent from './stores/models/cash-flow-event';
 
 configureMobX({ enforceActions: 'observed' });
 
 const App = () => (
-  <section className="my-money-calendar">
-    <Routes />
-  </section>
+  <StoreProvider>
+    <section className="my-money-calendar">
+      <Routes />
+    </section>
+  </StoreProvider>
 );
 
-window.idb = idb;
```

Comment:
This file can't merge as is. The stuff it's showing it will delete needs to be there. I would just check this file out from the base branch verbatim: `git checkout my-money-calendar -- cfgov/unprocessed/apps/mmt-my-money-calendar/js/index.js` and commit